### PR TITLE
Add Toggle Button component

### PR DESCRIPTION
**Pull request by ashwinimanoj**
I have created a base for Toggle Button and added the following props:
*label
*disabled

### CODE REVIEW

**Comment by ashwinimanoj** [comment]
> Why do we need wrap the component in injectIntl?

Toggle Button has a label property and well, it looks like I should have added formatting to it, I can do that, would that be fine?

**Comment by bsneha90** [comment]
Yes, that would be fine. Thanks.

File: ui/react-components/components/ToggleButton/ToggleButton.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { injectIntl } from "react-intl";
+import PropTypes from 'prop-types';
+
+const ToggleButton = (props) => {
+    const {label, disabled} = props;
+    return (
+            <label className={"toggle-btn-switch"}>
+                <input type="checkbox" disabled={disabled}/>
+                <span className={"toggle-btn-slider"} />
+                {label}
```
Comment (vinay-ThoughtWorks) [inline ui/react-components/components/ToggleButton/ToggleButton.jsx]: Span is self closed here. Label must be wrapped in span I guess. Please verify this.

Comment (ashwinimanoj) [reply ui/react-components/components/ToggleButton/ToggleButton.jsx]: I have removed label prop for now due to the css changes I have made.

**Comment by ashwinimanoj** [comment]
This is still a work in progress.

**Comment by ashwinimanoj** [comment]
> CSS is given within the code. Seems it can be extract to separate file.

@vinay-ThoughtWorks Are you talking about checkedColor? Well, I am setting a default hex value to checkedColor to make that color appear when the switch is toggled on and the color can be overridden by the parent. The background has to be set to change the color of the switch when it is toggled on and hence this approach, is this okay? If not, how could I make it better?

**Pull request merged**